tools/plugins/com.liferay.ide.server.tomcat.core/src/com/liferay/ide/server/tomcat/core/LiferayTomcatServer.java

```diff
@@ -409,6 +409,7 @@ public void setDefaults( IProgressMonitor monitor )
         setTestEnvironment( false );
         setDeployDirectory( ILiferayTomcatConstants.DEFAULT_DEPLOYDIR );
         setSaveSeparateContextFiles( false );
+        ServerUtil.setServerDefaultName(getServerWorkingCopy());
```

Comment:
Lets add a unit test that asserts this behavior.  Thanks!

Reply:
ok, I will add it.